otx/mpa/modules/datasets/cls_dir_dataset.py

```diff
@@ -33,7 +33,15 @@ class ClsDirDataset(BaseDataset):
         use_labels (bool): dataset with labels or unlabels
     """
 
-    def __init__(self, data_dir, pipeline=[], classes=[], new_classes=[], use_labels=True, **kwargs):
+    def __init__(
```

Comment:
Could you revert changes on this file? Even though it will be removed soon, but it can cause some confusion for other members.

Reply:
Ok. I just didn't want to spend time reverting this file as we don't use it anymore. But, I agree that it can lead to unnecessary changes in commits.